src/contradictory_claims/data/preprocess_cord.py

```diff
@@ -12,7 +19,7 @@ def filter_metadata_for_covid19(metadata_path: str, virus_lex_path: str, pub_dat
     :param metadata_path: path to CORD-19 metadata.csv file
     :param virus_lex_path: path to COVID-19 lexicon
     :param pub_date_cutoff: cut-off for publication date in the format 'yyyy-mm-dd'
-    :return: Metadata for filtered publications
+    :return: Dataframe of metadata for filtered publications
```

Comment:
I like your :return:s. I wonder if we should use static typing or if that's overkill. Hmmm. For now let's not, that's too fancy :P

Reply:
haha :p